diff --git a/dwl.c b/dwl.c
index bf340d8..151aa99 100644
--- a/dwl.c
+++ b/dwl.c
@@ -1613,15 +1613,9 @@ drawbar(Monitor *m)
 
 	if ((w = m->b.width - tw - x) > m->b.height) {
 		if (c) {
-			drwl_setscheme(m->drw, colors[m == selmon ? SchemeSel : SchemeNorm]);
-			drwl_text(m->drw, x, 0, w, m->b.height, m->lrpad / 2, client_get_title(c), 0);
-			if (c && c->isfloating)
-				drwl_rect(m->drw, x + boxs, boxs, boxw, boxw, 0, 0);
-		} else {
 			drwl_setscheme(m->drw, colors[SchemeNorm]);
 			drwl_rect(m->drw, x, 0, w, m->b.height, 1, 1);
 		}
-	}
 
 	wlr_scene_buffer_set_dest_size(m->scene_buffer,
 		m->b.real_width, m->b.real_height);

